### PR TITLE
my first project

### PR DESCRIPTION
**Pull request by UnknownKira**

### CODE REVIEW

**Comment by ValeeraCatcha** [comment]
approved

**Comment by ValeeraCatcha** [comment]
approved

File: HT 1.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+	<head>
```
Comment (egemon) [inline HT 1.html:3]: in general I prefer two spaces as tab width. Try it.

File: HT 1.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+					<p>Хочу учить front-end потому-что:</p>
+					<ul class="list">
+						<li class="list-item">Я поклялся служить Нерзулу</li>
+						<li class="list-item">Нужно больше золота</li>
+						<li class="list-item">Нужно построить Зиккурат</li>
+					</ul>
+				</div>
+				<div class="content border">
+					<p>Мой контактный телефон</p>
+					<p id="indent">666666666</p>
```
Comment (egemon) [inline HT 1.html:26]: don't use id for styling only for JS or input labels

File: css/reset.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
```
Comment (egemon) [inline css/reset.css:6]: good, take a look at CDN about style reseting. Maybe you will like it.

**Review by egemon — APPROVED** [review]

**Pull request merged**